static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -331,13 +338,38 @@ export function MetricsExtractionRuleForm({
                   )}
                 />
               }
-              placeholder={t('Select span attribute')}
-              creatable
-              formatCreateLabel={value => `Custom: "${value}"`}
               onCreateOption={value => {
                 setCustomAttributes(curr => [...curr, value]);
                 model.setValue('spanAttribute', value);
               }}
+              components={{
+                // Replaces the dropdown options
+                Option: containerProps => {
+                  if (containerProps.data.__isNew__) {
+                    return (
+                      <Fragment>
+                        <OptionInfo>
+                          {tct(
+                            'Select an attribute or create one. [link:See how to instrument a custom attribute.]',
+                            {
+                              link: (
+                                <ExternalLink href="https://docs.sentry.io/product/explore/metrics/metrics-set-up/" />
```

Comment:
I don't think we should be using this link, or maybe the docs should be updated, because currently old `metrics` API is mentioned on those docs, and we are trying to move away from it. There is no mention here about new span based attributes, or attaching an attribute to a span

Reply:
Afaik @ale-cota is working on updating this doc

Reply:
Nice, ok, got it. Should we wait with this PR until the docs are updated, maybe we can link the specific section of those docs?

Reply:
yes, I think we should but I will leave it up to @ale-cota  🙂

Reply:
Perfect, thank you :)

Reply:
We don't have to wait for the docs update to merge this, since it's behind a feature flag and only available to Sentry internal. Thanks!